src/main/scala/tech/cryptonomic/conseil/tezos/TezosDatabaseOperations.scala

```diff
@@ -218,6 +218,32 @@ object TezosDatabaseOperations extends LazyLogging {
   def getAccountsCheckpointSize(): DBIO[Int] =
     Tables.AccountsCheckpoint.distinctOn(_.accountId).length.result
 
+  /**
+    * Takes all existing account ids and puts them in the
+    * checkpoint to be later reloaded, based on the passed block reference
+    */
+  def refillAccountsCheckpointFromExisting(hash: BlockHash, level: Int, timestamp: Instant)(
+      implicit ec: ExecutionContext
+  ): DBIO[Option[Int]] = {
+    logger.info("Relaoding all ids for existing accounts and checkpointing with block hash {}, level {} and time {}",
```

Comment:
Could we change this to the following?

"Fetching all ids for existing accounts and checkpointing them with block hash {}, level {} and time {}"

Reply:
done